int32 relay_number
bool state
---
bool status
string message